package.json

```diff
@@ -77,6 +77,20 @@
     "lint:fix:cpp": "find src -name \"*.?pp\" | xargs clang-format -i",
     "postinstall": "node scripts/submit-analytics.js"
   },
+  "workspaces": [
```

Comment:
Is it a requirement that we manually maintain the list of workspaces?

Reply:
Yeah. We can use wildcards here, but scripts (ex `npm run build --workspaces`) are ran on the sub-packages in the order they appear in this list, so for things like the `realm-network-transport` where some packages are depending on it being built before they can build themselves, we need to be explicit on this order.

This is a drawback to how Lerna worked, where packages would be installed based on their dependency ordering (any package depending on "package A" would be installed (hence built) after "package A").